src/main/java/org/sasanlabs/internal/utility/annotations/VulnerabilityLevel.java

```diff
@@ -35,6 +35,33 @@
      */
     String htmlTemplate() default "";
 
+    /**
+     * This information can be used by Scanners to know location in request where payload can be
+     * injected. Default value is {@link RequestParameterLocation#QUERY_PARAM}
+     *
+     * @return location of parameter
+     */
+    RequestParameterLocation location() default RequestParameterLocation.QUERY_PARAM;
+
+    /**
+     * This information can be used by Scanners to know the name of the key whose value is read by
+     * the endpoint/vulnerability level for performing the operation.
+     *
+     * @return name of the parameter which holds the value
+     */
+    String parameterName() default "";
```

Comment:
Assumption is only one parameter will correspond to one Level. Not sure if there is any issue. Will tackle if so in future.